buildspec-ecr-upload.yml

```diff
@@ -0,0 +1,12 @@
+version: 0.2
+
+phases:
+  pre_build:
+    commands:
+      - echo Logging in to Amazon ECR...
+      - aws --version
+      - docker --version
```

Comment:
I'd recommend using `docker version` instead of `docker --version` so you get information about the server version as well.